scripts/variantstore/README.md

```diff
@@ -16,9 +38,9 @@ After using the json file to auth as that service account, you can query the
     bq --project_id <project> query --use_legacy_sql=false 'select sum(total_bytes_billed) as total from `region-us`.INFORMATION_SCHEMA.JOBS_BY_USER where creation_time > "2021-04-16 00:00:00" and creation_time < "2021-04-18 00:00:00" and total_bytes_billed > 1 limit 100'
 
 
-### Running the VAT pipeline
+## Running the VAT pipeline
 The VAT pipeline is a set of WDLs
-GvsSitesOnlyVCF.wdl
-GvsValidateVAT.wdl
+ - GvsSitesOnlyVCF.wdl
+ - GvsValidateVAT.wdl
 
-The pipeline takes in a jointVCF and outputs a table in BigQuery
+The pipeline takes in a jointVCF and outputs a table in BigQuery.
```

Comment:
I don't know this process, @RoriCremer ?

Reply:
@kbergin I checked with @RoriCremer and the VAT process isn't yet at a point where it can be documented for those outside the team, so I think we should wait to do this in another PR.